### DISCUSSION TITLE
VoiceCall Record created for Voicemail Call?

### DISCUSSION THREAD

**Discussion by jlaurie87** [amazon-connect/amazon-connect-salesforce-scv#112] [Q&A] [answered]
Bit of an odd one. 

I am curious if the VE2 app creates a VoiceCall Record as part of its deployment. 
I have tested deleting the voicecall record made when a call comes in, if the call is routed to voicemail. 

The record deletes fine, but it seems like VE2 is making the record again a few minutes later. I just wanted to check if that was the case or if I am missing something?

**Comment by pagmax** [comment]
I am facing an issue where in if the phone number is dialled it always rings first and then says that "The call cannot be completed. Please try again". Any insights on this?

**Reply by dcarbonetti8squad** [reply]
this should be raised seperately

**Comment by dcarbonetti8squad** [comment, marked as answer]
if you are using service cloud voice, there is a lambda called CTRDataSync that synchronises call data for all calls that happen in amazon connect into a salesforce voicecall record; that is likely what is causing it. the service cloud voice group is a better place to discuss that : https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000Gw6sSAC?tab=discussion

**Reply by dougjaso** [reply]
Correct. VMX itself does not incorporate the creation of the VoiceCall object as the solution is designed to work in a myriad of configurations, not just Service Cloud Voice. The CTR Sync function is most likely your culprit, as it has been modified to create VoiceCall records for voice calls that do not have them.
